torch/testing/_internal/common_modules.py

```diff
@@ -376,6 +393,36 @@ def module_inputs_torch_nn_Embedding(module_info, device, dtype, requires_grad,
     ]
 
 
+def module_inputs_torch_nn_MultiheadAttention(module_info, device, dtype, requires_grad, **kwargs):
+    make_input = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    samples = []
+    bool_vals = (True, False)
```

Comment:
Can you please add a comment before this section indicating that these generated samples are for validating no-batch-dim support?

In the future, we'll want to expand this function to include other samples as well (not for this PR).